src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -0,0 +1,201 @@
+@import "../../patternfly-imports";
+
+.pf-c-options-menu {
+  // Toggle
+  --pf-c-options-menu__toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-options-menu__toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-options-menu__toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-options-menu__toggle--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-options-menu__toggle--MinWidth: var(--pf-global--target-size--MinWidth);
+  --pf-c-options-menu__toggle--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-options-menu__toggle--BorderTopColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-options-menu__toggle--BorderRightColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-options-menu__toggle--BorderBottomColor: var(--pf-global--BorderColor--dark);
+  --pf-c-options-menu__toggle--BorderLeftColor: var(--pf-global--BorderColor--light-200);
+  --pf-c-options-menu__toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-options-menu__toggle--active--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-options-menu__toggle--active--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-options-menu__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-options-menu__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
+  --pf-c-options-menu__toggle--m-plain--BorderColor: transparent;
+  --pf-c-options-menu__toggle--m-plain--Color: var(--pf-global--Color--200);
+  --pf-c-options-menu__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
+
+  // Toggle Arrow
+  --pf-c-options-menu__toggle-icon--MarginLeft: var(--pf-global--spacer--md);
+
+  // Menu
+  --pf-c-options-menu__menu--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-options-menu__menu--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-options-menu__menu--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-options-menu__menu--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-options-menu__menu--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-options-menu__menu--Top: calc(100% + var(--pf-global--spacer--xs));
+  --pf-c-options-menu__menu--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-options-menu--m-top__menu--Top: 0;
+  --pf-c-options-menu--m-top__menu--Transform: translateY(calc(-100% - var(--pf-global--spacer--xs))); // The "dropup" menu must be transformed up and this calculates how much to create space between the toggle and menu
+
+  // Menu Item
+  --pf-c-options-menu__menu-item--PaddingTop: var(--pf-global--spacer--sm);
```

Comment:
The spacing looks too small between the spacer and list item above and below it. Did you follow these spacing guidelines? 
<img width="354" alt="screen shot 2019-01-21 at 11 00 21 am" src="https://user-images.githubusercontent.com/20118816/51485386-e740ca80-1d6b-11e9-851d-896643b9c58d.png">

Reply:
Oh I'm just using the dropdown's spacing. I'm assuming this should match the dropdown? Or should the dropdown (and select) menu item spacing be changed you think?

Reply:
@gdoyle1 can you double check this

Reply:
@mcoker @christiemolloy It should match the dropdown spacing but I think the space between the spacer and list item needs to be increased in general. Could we do that?

Reply:
@gdoyle1 yep, just let me know what you'd like it to be.

Reply:
@mcoker 16 px below "size" then the divider line then 16 px above "ascending" please!

Reply:
thanks @gdoyle1 :) Is this just for this type of menu? or is this for all dropdowns with a separator? I copied these styles from the dropdown - https://pf-next.com/components/Dropdown/examples/

Reply:
@mcoker For all dropdowns with a separator. Thank you!

Reply:
Awesome, thanks. Updated options-menu, dropdown, and select.

Reply:
@mcoker That spacing looks a lot better! Thank you